scripts/create_baseline_stubs.py

```diff
@@ -160,6 +165,7 @@ def main() -> None:
     run_stubdefaulter(stub_dir)
 
     run_isort(stub_dir)
+    run_ruff(stub_dir)
```

Comment:
```suggestion
    run_ruff(stub_dir)
    run_isort(stub_dir)
```

I think that we should call `ruff` as the first one. Because it change semantics, not only style.

By the way, `isort` can also be later replaced with `ruff`.

Reply:
Oh, I see https://github.com/python/typeshed/pull/10458#issuecomment-1642008989